Frameworks/UIKit/UITextView.mm

```diff
@@ -759,7 +763,7 @@ - (CGSize)sizeThatFits:(CGSize)fitSize {
     centerRect.origin.y = 0;
     centerRect.size = fontExtent;
     // TODO(DH)
-    //EbrCenterTextInRectVertically(&centerRect, &fontExtent, _font);
+    // EbrCenterTextInRectVertically(&centerRect, &fontExtent, _font);
```

Comment:
This might not be necessary, but I'll add an issue to track "something about vertical text centering in UITextField"

This is more of a CT concern, but EbrCenterTextInRectVertically was part of CGContext for some unknown reason.